charts/airbyte-worker/templates/deployment.yaml

```diff
@@ -10,6 +10,8 @@ metadata:
     app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
     app.kubernetes.io/managed-by: {{ .Release.Service }}
     app.airbyte.io/fullname: {{ include "airbyte.fullname" . }}
+  annotations:
+    helm.sh/hook: post-install,post-upgrade
```

Comment:
why do we need this?

Reply:
This one leftover from testing, removed.